CMakeLists.txt

```diff
@@ -13,29 +13,35 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake/Modules")
 ### HunterGate and cache ###
 ############################
 
+option(DRISHTI_USE_DRISHTI_UPLOAD "Use drishti upload configuration" OFF)
 set(drishti_upload_init "${CMAKE_CURRENT_LIST_DIR}/drishti-upload/init.cmake")
 
-if(EXISTS "${drishti_upload_init}")
-  include("cmake/HunterGate.cmake") # HUNTER_ENABLED
-
+if(EXISTS "${drishti_upload_init}" AND DRISHTI_USE_DRISHTI_UPLOAD)
```

Comment:
I think combination when `DRISHTI_USE_DRISHTI_UPLOAD=ON` and `${drishti_upload_init}` not exist should be handled as an error.

Reply:
That's a good precaution.  I'll rework the logic to support an error message under those conditions while still allowing the local repository build via `cmake/Hunter/config.cmake`.